src/sentry/api/endpoints/organization_details.py

```diff
@@ -271,6 +295,16 @@ def validate(self, attrs):
                 raise serializers.ValidationError(
                     {"avatarType": "Cannot set avatarType to upload without avatar"}
                 )
+        organization = self.context["organization"]
+        has_api_auth_provider = features.has("organizations:api-auth-provider", organization)
```

Comment:
Nit: I don't see any purpose in `organization = self.context["organization"]` if we're not going to reuse it.

Nit: We're using `has_api_auth_provider` for the third time here, perhaps it could be a class property?

Reply:
Ok removed and added as class property